### ISSUE TITLE
Gather preliminary research on our target user (foodseekers) by conducting a user insights survey.

### ISSUE THREAD

**Issue by naraedesigns** [hackforla/food-oasis#1525]

**Comment by GigiUxR** [comment]
I have provided feedback via slack. One major hurdle with a survey of this length will be recruitment. I am suggesting a recruitment service like Prolific through which we would be able to pre-screen our sample for low-income and other relevant demographics . I pulled a quick quote from them that is between $70 to $100 depending on questionnaire time # of respondents. Alternatively, we could run a raffle for $50 but would need to clean our data to eliminate speeders since the allure of the prize will attract those interested only in the chance of winning. Would we have funds already available for this sort of work? Would we need to crowd-source funds?

**Comment by staceyrebekahscott** [comment]
As per the Design team discussion on 01/26, James suggested that we send the survey to the organizations that we have listed in our database. I suggested that we reach out to the ones who have been validated recently. Erin should be able to provide guidance on this via the work with the UCLA students.

**Comment by GigiUxR** [comment]
@staceyrebekahscott please clarify -- food resource organizations would distribute the survey to their incoming food seekers?

**Comment by staceyrebekahscott** [comment]
@GigiUxR No, the idea is that these organizations would have representatives that might know how to answer these questions on behalf of food seekers. It is unlikely that we would be able to get direct contact with food seekers through this method. 

We do not have any funds we can allocate for using sites like Prolific. We would need to find something we could use for free. The grant money we received recently likely went to ongoing expenses such as web hosting and domain purchases.

**Comment by GigiUxR** [comment]
( Some of the questions being asked in the survey will be close to impossible for anyone other than the seeker to answer. )

**Comment by naraedesigns** [comment]
Old notes:

I informed Bryan, Sarah, and Jelena on 1/21 about the survey. Bryan reviewed the survey and provided helpful feedback. Gigi reviewed survey on 1/25 and provided helpful feedback. 

In my conversation with Gigi and Bryan, the possibility of sharing the survey on Reddit came up, and Gigi noted there are some restrictions but will look into it.

**Comment by naraedesigns** [comment]
### Sharing out the survey

1/26/23 - I launched the survey via the following:
- Slack communities
- Twitter
- Instagram
- My personal contacts in social work
- I also shared about the survey in the design team meeting and requested help to share out the survey 

1/27/23 - I shared the survey 
- LinkedIn
- Email

**Comment by naraedesigns** [comment]
### 1/26/23 Design team meeting

I shared about the survey during the 1/26/23 design team meeting and received positive feedback and helpful suggestions. 

- James suggested sharing the survey with organizations listed in our directory. Stacey noted it’d be best to connect food resource providers that have been updated recently and that Erin would be a good person to coordinate with. 
- Natalie noted the survey looks great, and suggested a Spanish version of the survey. I agreed that would be useful - added to the list of items to do.
- I noted I would create a new slack channel so people can follow along for updates / interesting updates and share anything they discover too.

**Comment by fancyham** [comment]
This is great — let’s see what kinds of responses we get from various places.  Using our own networks might not be best but that’s part of what we’re learning.  If we can reach out to people in the food distribution field, they can also give recommendations.

I recommend taking notes on the project plan about when and where the survey was distributed, what responses we got, and a change log of survey changes (as it may evolve over time).  Those will be useful when processing the results later.

I’d also recommend putting our URL on the thank-you page (or even on the intro page) so that they can trust us.

These surveys could also be done in-person at food distribution sites (or really, anywhere) with guerrilla-style research — the survey forms can also be printed for people to fill out by hand and hand-entered later:

If doing this in person, one should:
- Get permission and consent from both food distributor, participants, etc.
- Ideally, give some compensation for their time as a thank-you (even some chocolates or chips or something are a nice gesture)
- Make it obvious who you are and why you’re doing it (i.e. that you’re a volunteer for a non-profit website, not the INS, for example). A sign can help.
- Be prepared to give out info about foodoasis.la to those curious - handouts are nice.
- Approach this with a trauma-informed mindset and to make sure you avoid [deficit-model framing](https://www.google.com/search?q=deficit-model) of yourself and the respondent
- Be sure to record in the data where, when this survey is being taken (applies to everything) because that may be relevant later when processing the data.


@naraedesigns, FYI, our Slack instance doesn’t keep things older than 90 days, so if there’s anything posted in the slack channel #fola-user-insights that are relevant for the future, please copy and paste it in this issue’s Github comments (which are permanent)

**Comment by naraedesigns** [comment]
1/28 - Bryan's comments via slack

Reddit might not be the perfect place to do the test due to forum restrictions but one way to approach it is to post in regional forums like LA and explain that it’s a quick anonymous study, and also to ask ‘where should I ask this’?

I agree that it’s best to recruit people and pay for their time — for my previous guerrilla studies, we’d give like a $20 gift certificate to a market (cash is even better, but we used gift cards so we could expense it (and to minimize embezzlement risk (researcher walking away with money and faking results))

The funds we’ve talked about in the past is just out of our (perhaps my :) pocket — so yes, we can discuss amounts if we need to recruit from a service.  Another way might be to do a Mechanical Turk — I’ve never done it before but it might be worth checking out: http://mturk.com/. Also, other sites: https://www.google.com/search?q=mturk+vs&ie=UTF-8&oe=UTF-8&hl=en&client=safari. 

One other way is to just go to a site in person and do the survey with someone (with a honorarium/thank-you gift), or just asking people if you could ask them.  That’s probably an excellent way and arguably more specific and rigorous (‘these are survey results at this location on this date’ vs anonymous self-reported — different methods, different strengths).

If folks are doing it in person, gotta get consent from the service provider, too (you both probably know this, but in case, other FOLAers try it out) and will want some additional protocols so that we don’t look like INS or anything.

**Comment by naraedesigns** [comment]
A friend also suggested the following:
"You could find mutual aid groups in LA and ask them to pass along the survey. If they can’t share the survey, maybe they can do Subject Matter Expert (SME) interviews where they can share their POV as a provider of mutual aid/free food resources."

**Comment by naraedesigns** [comment]
@fancyham just a heads up that I created a [spreadsheet](https://docs.google.com/spreadsheets/d/15cBLfCXiO04B0cYxr-XVUucKe90iu3qZSWle1v53Thg/edit#gid=0) that includes our ideas for survey distribution as well as what's been done already. I also added this info as 2 separate links for easy viewing to the research plan, and to the main blurb of this issue. 

Re: change log suggestion
I added a "change log" section on the research plan and noted that I included the URL on the intro blurb of the survey.

**Comment by naraedesigns** [comment]
### 1/30/23 Meeting

Participants: Gigi, Bryan, myself 

Gigi, Bryan, and I met tonight and discussed possible ways to collaborate - we plan to meet again along with Sarah and Jelena to continue the conversation.

Action items for me
- [ ] - Create a group slack so we have a set place to communicate altogether
- [ ] - Share my availability for a meeting 

Action items for the group 
- [ ] Schedule a meeting to discuss next steps (Gigi, Bryan, Sarah, Jelena, myself)

**Comment by naraedesigns** [comment]
I prepared a quick [consent form (https://docs.google.com/document/d/1RNT7q3jVEifhsE1QM0NDG8Sr_yDgRWZSotjX1i38dWY/edit?usp=sharing) and [intro deck](https://docs.google.com/presentation/d/1V_IS9LPxSRHCpuYavv8XT3vE-zF-nefc2EDabamkuus/edit?usp=sharing ) for the research team, based on yesterday's conversation with Bryan and Gigi. 

I also shared both documents into the slack group chat with Gigi, Bryan, Sarah, Jelena, and myself.

**Comment by narae-ux** [comment]
Updates on Survey (V1)

- One-week checkpoint - On 2/2, the survey had 4 responses.
- Two-week checkpoint - On 2/9, the survey still had 4 responses.
- On 2/16, I closed the survey to new responses.

*Distribution strategy was not changed and active distribution efforts were paused after the 2/1 meeting, as the group decided to further edit the current survey.

**Comment by narae-ux** [comment]
I messaged the team currently working on this issue: Gigi, Bryan, Sarah, Jelena, Shirley, Jordan. 

I noted that the team has done a great job executing further on the survey, and that I will be transitioning off this issue in order to focus on design issues from now on. I noted that I can help answer any questions as needed and that I look forward to seeing the final deliverables/results of our work. I also updated the #fola-user-insights channel with this information. 

I also messaged new members Shirley and Jordan to introduce myself and noted I can help answer questions as needed.

**Comment by fancyham** [comment]
The new pop-up survey tool (in progress) can help us use this do this survey again (or a derivative of it) 
- #1769

**Comment by SAUMILDHANKAR** [comment]
A lot of issues about surveys of food seekers have already been closed (
- #2483, -#2382, -#2362, -#2250, -#1769, -#1655, -#965, -#892) And a lot of issues about surveying food seekers are being planned (-#2373, -#2071, -#1525, -#1472, -#1441, -#1379). And yet we still do not have survey results from food seekers which are reliable and statistically significant in number. 


Perhaps since we do not have a lot of website users, we can not have people who are primarily food seekers filling up the surveys and it is like a catch 22 problem. Personally, I agree that it is hard to replace the information that we need from primary food seekers with any other kind of survey. Providing incentives for the survey may also be defeating the purpose a little. Ideally, if we're helping people use free food sources in some way then they should be able to tell us where they need more help, but it's hard for us to reach them through our partners or listings. One question I want answered is that if people are using free food sources then why are they not using our website? 


Maybe until we have a lot of website users, we can have research team focus on secondary research instead of trying to get survey filled up from primary users. So, we can icebox issues related to reaching primary food seekers until website visitors improve and meantime do secondary research and prioritize all issues listed above accordingly.
